queries.sql

```diff
@@ -1 +1,7 @@
+SELECT * FROM animals WHERE name LIKE '%mon%';
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-13';
+SELECT name FROM animals WHERE neutered = true AND escape_attempts < 3;
+SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
+SELECT * FROM animals WHERE name <> 'Gabumon';
```

Comment:
- [ ] You missed two major queries here. Find the requirement screenshot below, and implement accordingly 👍 

## Missed query:
- ![image](https://github.com/david30-maker/animal-database/assets/50990925/522de1c0-1d0c-4cee-90ba-9a8378dbc912)
-  ![image](https://github.com/david30-maker/animal-database/assets/50990925/d9ba2574-9bbb-44cc-a758-1720807a0a40)